zerver/lib/actions.py

```diff
@@ -1834,7 +1834,6 @@ def check_send_typing_notification(sender: UserProfile, notification_to: Union[S
         operator=operator,
     )
 
-
```

Comment:
Stray newline. :)

Reply:
Oh, I deleted this because there was 2 blank lines between this method and the one below, and isn't Zulip convention to separate methods with one blank line?